### DISCUSSION TITLE
Moving the `Save` button on a custom page

### DISCUSSION THREAD

**Discussion by bilogic** [filamentphp/filament#8341] [Help]
![image](https://github.com/filamentphp/filament/assets/946010/78447fd6-6fed-4812-990b-e62bd9befa3c)

Is there a way to move the Save changes button to where the arrow is pointing?

**Comment by leandrocfe** [comment]
If you have access the view
```html
<div class="flex justify-end">
    <x-filament::button>Save changes</x-filament::button>
</div>
```

**Reply by bilogic** [reply]
thanks, I saw this https://filamentphp.com/docs/3.x/forms/actions#controlling-the-horizontal-alignment-of-independent-form-actions

If I added this method for my page, what should I do inside the blade?
```PHP
    protected function getFormActions($form)
    {
        Actions::make([
            Action::make($form.'UpdateAction')
                ->label(__('filament-panels::pages/auth/edit-profile.form.actions.save.label'))
                ->submit($form),
        ])->alignment(Alignment::end);
    }
```
